session.js

```diff
@@ -0,0 +1,111 @@
+/*!
+ * diva-session
+ * Copyright(c) 2017 Alliander, Koen van Ingen, Timen Olthof
+ * BSD 3-Clause License
+ */
+
+const divaStateModule = require('./diva-state');
```

Comment:
Ideally this should not depend on diva-state. But since we want to store both (optionally) in redis, it might be better to leave the dependency here for now.

Reply:
As discussed, lets refactor further later.